src/scripts/install-docker-credential-helper.sh

```diff
@@ -55,17 +58,28 @@ echo "Downloading credential helper $HELPER_FILENAME"
 BIN_PATH="/usr/local/bin"
 mkdir -p "$BIN_PATH"
 RELEASE_TAG="$PARAM_RELEASE_TAG"
-RELEASE_VERSION=$(curl -Ls --fail --retry 3 -o /dev/null -w '%{url_effective}' "https://github.com/docker/docker-credential-helpers/releases/latest" | sed 's:.*/::')
+base_url="https://github.com/docker/docker-credential-helpers/releases"
+RELEASE_VERSION=$(curl -Ls --fail --retry 3 -o /dev/null -w '%{url_effective}' "$base_url/latest" | sed 's:.*/::')
 if [ -n "${RELEASE_TAG}" ]; then
   RELEASE_VERSION="${RELEASE_TAG}"
 fi
-DOWNLOAD_URL="https://github.com/docker/docker-credential-helpers/releases/download/${RELEASE_VERSION}/${HELPER_FILENAME}-${RELEASE_VERSION}-amd64.tar.gz"
 
-echo "Downloading from url: $DOWNLOAD_URL"
-curl -L -o "${HELPER_FILENAME}_archive" "$DOWNLOAD_URL"
-tar xvf "./${HELPER_FILENAME}_archive"
-chmod +x "./$HELPER_FILENAME"
+# Starting from v0.7.0, the release file name is changed to docker-credential-<helper-name>-<os>-<arch>
```

Comment:
Fyi, should be `docker-credential-<helper-name>-<os>-<arch><variant>`

Reply:
@EricRibeiro

Reply:
@crazy-max, for my understanding, what would be the variant in the string below?

```
docker-credential-osxkeychain-v0.7.0.darwin-amd64
```

It looks like `docker-credential-<helper-name>-<version>.<os>-<arch>` 🤔

@bjohnso5, I believe they are referring to my comment. This shouldn't have any functional impact on the orb (if that is your concern).

Reply:
This would refer for example to `arm` variants like `v6` and therefore would be `armv6` in filename.

There is also [`amd64` variants ](https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels) but we don't produce binaries for them atm (`v2`, `v3`, `v4`).

Reply:
Not atm, this variant is really early. Will keep you posted anyway :+1:

Reply:
I appreciate it. I'll update the comment to reflect your input.